runtime/compiler/env/VMJ9.h

```diff
@@ -1544,8 +1544,6 @@ class TR_J9SharedCacheVM : public TR_J9VM
    virtual bool               sameClassLoaders(TR_OpaqueClassBlock *, TR_OpaqueClassBlock *);
    virtual bool               isUnloadAssumptionRequired(TR_OpaqueClassBlock *, TR_ResolvedMethod *);
    virtual bool               classHasBeenExtended(TR_OpaqueClassBlock *);
-   virtual bool               isGetImplInliningSupported();
-   virtual bool               isGetImplAndRefersToInliningSupported();
```

Comment:
You should also remove these queries in `TR_J9SharedCacheServerVM` if it's being removed in `TR_J9SharedCacheVM`.

Reply:
Fixed in commit b03ac952857a54bfc3169e631a9d866990949f81